specification/storagecache/resource-manager/Microsoft.StorageCache/stable/2020-10-01/storagecache.json

```diff
@@ -1359,9 +1361,249 @@
+        "usernameSource": {
+          "description": "This setting determines how the system gets username and group names for clients.",
+          "enum": [
+            "AD",
+            "LDAP",
+            "File",
+            "None"
+          ],
+          "x-ms-enum": {
+            "name": "UsernameSource",
+            "modelAsString": true
+          },
+          "type": "string",
+          "default": "None"
+        },
+        "groupFileURI": {
+          "type": "string",
+          "description": "The URI of the file containing the group (etc/group) information. This field must be populated when 'usernameSource' is set to 'File'."
+        },
+        "userFileURI": {
+          "type": "string",
+          "description": "The URI of the file containing the user (etc/passwd) information. This field must be populated when 'usernameSource' is set to 'File'."
+        },
+        "ldapServer": {
+          "type": "string",
+          "description": "The fully qualified domain name or IP address of the LDAP server to use."
+        },
+        "ldapBaseDn": {
+          "type": "string",
+          "description": "The base distinguished name for the LDAP domain."
+        },
+        "encryptLdapConnection": {
+          "type": "boolean",
+          "description": "This indicates if the LDAP connection should be encrypted."
+        },
+        "requireValidCertificate": {
           "type": "boolean",
-          "description": "root squash of cache property."
+          "description": "Determines if the certificates should be validated by a certificate authority. When true caCertificateURI must be provided."
+        },
+        "autoDownloadCertificate": {
+          "type": "boolean",
+          "description": "Determines if the certificate should be automatically downloaded. This applies to 'caCertificateURI' when 'requireValidCertificate' is true, or a self signed certificate otherwise."
+        },
+        "caCertificateURI": {
+          "type": "string",
+          "description": "The URI of the CA certificate to validate the LDAP secure connection. This field must be populated when 'requireValidCertificate' is set to 'true'."
+        },
+        "usernamePolled": {
+          "description": "Indicates if the HPC Cache has performed the username download successfully.",
+          "type": "string",
+          "readOnly": true,
+          "enum": [
+            "Yes",
+            "No",
+            "Error"
+          ],
+          "x-ms-enum": {
+            "name": "UsernamePolledType",
+            "modelAsString": true
+          }
+        },
+        "credentials": {
+          "description": "When present, these are the credentials for secure LDAP connection.",
+          "properties": {
+            "bindDn": {
+              "type": "string",
+              "description": "The Bind Distinguished Name identity to be used in the secure LDAP connection. This value is stored encrypted and not returned on response."
```

Comment:
```suggestion
              "description": "The Bind distinguished name identity to be used in the secure LDAP connection. This value is stored encrypted and not returned on response."
```

Reply:
Or capitalize "password" in `bindPassword`'s description below. Either way is fine. Just a consistency issue.

Reply:
Fixed in next commit.

Reply:
Fixed in next commit.